buoy_real_size: 0.1